### PR TITLE
Fixes 459: Extends find_units() for neuron parameters

### PR DESCRIPTION
**Pull request by appukuttan-shailesh**
**Addresses issue #459** 
find_units() should now return units for both:
- recordable variables, and
- neuron parameters

for all standard models

### CODE REVIEW

**Comment by apdavison** [comment]
Very nice. Could you also update the docstring for `find_units()`, and check any mentions there might be in the documentation?

**Comment by appukuttan-shailesh** [comment]
`find_units()` did not have any docstring, so added it to both Population and Assembly.

File: pyNN/common/populations.py [hunk 1]
```diff
@@ -405,6 +405,10 @@ def initialize(self, **initial_values):
             self.initial_values[variable] = initial_value
 
     def find_units(self, variable):
+        """
+        Returns units of the specified variable. Works for all the recordable
```
Comment (apdavison) [inline pyNN/common/populations.py]: "Returns units of the specified variable or parameter, as a string"

Comment (apdavison) [inline pyNN/common/populations.py]: In case people think it returns a Quantities units object

Comment (appukuttan-shailesh) [inline pyNN/common/populations.py]: Yes, changed that now.

**Pull request merged**